src/components/validations/with-validation.hoc.js

```diff
@@ -260,6 +279,34 @@ const withValidation = (WrappedComponent) => {
     }
   }
 
+  WithValidation.contextType = ValidationsContext;
+
+  WithValidation.propTypes = {
+    children: PropTypes.node, // Children elements
+    name: PropTypes.string.isRequired, // Name to uniquely identify the component
+    value: PropTypes.string, // The current value of the component
+    onBlur: PropTypes.func, // Custom function to be called when the component blurs
+    onChange: PropTypes.func, // Custom function called when component value changes
+    ...Object.values(VALIDATION_TYPES).reduce((acc, type) => ({
+      ...acc,
+      [type]: PropTypes.oneOfType([ // The info validations that should be run against the value
+        PropTypes.func,
+        PropTypes.arrayOf(
+          PropTypes.shape({
+            message: PropTypes.func,
+            validate: PropTypes.func
+          })
+        ),
+        PropTypes.arrayOf(PropTypes.func)
+      ])
+    }), {})
+  };
+
+  WithValidation.defaultProps = Object.values(VALIDATION_TYPES).reduce((acc, type) => ({
+    ...acc,
+    [type]: []
+  }), {});
+
```

Comment:
this code already exists as static properties

Reply:
Yeah I've missed that when I merged in the conflicts, will get it removed